### DISCUSSION TITLE
Variaveis de componentes laravel 8 não funcionam no linux

### DISCUSSION THREAD

**Discussion by jhonatasFelipe** [laravel/framework#36095] [Q&A] [answered]
Eu implementei um componente no laravel 8 usando o windows , tudo funcionou perfeitamente , mas quando subo os arquivos para o servidor linux as variaveis de componente que eu não injetei no construtor não rebem valor nenhum ocasionando um erro. 
Alguém sabe me dizer oque pode estar acontecendo ?

Essa é view do componente
![blade componente](https://user-images.githubusercontent.com/8534062/106341700-f58c2000-627c-11eb-9e4f-7281bbaf2a91.PNG)
essa é a classe do componente. 
![classe componente](https://user-images.githubusercontent.com/8534062/106341702-f624b680-627c-11eb-8012-e8cdadda78c4.PNG)
Esse é o erro que qpqerece no servidor linux
![erro laravel linux](https://user-images.githubusercontent.com/8534062/106341704-f6bd4d00-627c-11eb-9504-3851874108f5.PNG)

**Comment by L4ZARIN3** [comment, marked as answer]
to tendo esse problema tbm e ninguem sabe como solucionar, o unico jeito e criando variaveis globais no AppServiceProvider, mais ai e gambiarra

**Comment by L4ZARIN3** [comment]
e so pra constar se vc der um dd() no windows ele vai rodar o output do dd() tranquilamente, mais no linux o dd() nao vai dar output, entao chego a conclusão de que nem a classe do componente esta sendo chamada no linux.
